#! /bin/sh /usr/share/dpatch/dpatch-run
## 96_gcc4.3-includes.dpatch by Tobias Grimm <tg@e-tobi.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes FTBFS for GCC 4.3.

@DPATCH@
diff -urNad vdr-plugin-burn-0.1.0~pre21~/adaptor.h vdr-plugin-burn-0.1.0~pre21/adaptor.h
--- vdr-plugin-burn-0.1.0~pre21~/adaptor.h	2006-09-16 20:33:36.000000000 +0200
+++ vdr-plugin-burn-0.1.0~pre21/adaptor.h	2008-06-10 20:08:12.000000000 +0200
@@ -11,6 +11,7 @@
 #include <iterator>
 #include <vdr/epg.h>
 #include <vdr/tools.h>
+#include <algorithm>
 
 namespace vdr_burn
 {
diff -urNad vdr-plugin-burn-0.1.0~pre21~/filter.h vdr-plugin-burn-0.1.0~pre21/filter.h
--- vdr-plugin-burn-0.1.0~pre21~/filter.h	2006-09-16 20:33:36.000000000 +0200
+++ vdr-plugin-burn-0.1.0~pre21/filter.h	2008-06-10 20:08:12.000000000 +0200
@@ -10,6 +10,7 @@
 
 #include "common.h"
 #include <iterator>
+#include <algorithm>
 
 namespace vdr_burn
 {
diff -urNad vdr-plugin-burn-0.1.0~pre21~/iconvwrapper.c vdr-plugin-burn-0.1.0~pre21/iconvwrapper.c
--- vdr-plugin-burn-0.1.0~pre21~/iconvwrapper.c	2006-09-16 20:33:36.000000000 +0200
+++ vdr-plugin-burn-0.1.0~pre21/iconvwrapper.c	2008-06-10 20:08:12.000000000 +0200
@@ -8,6 +8,8 @@
 #include "iconvwrapper.h"
 #include <cerrno>
 #include <sstream>
+#include <string.h>
+#include <stdlib.h>
 
 namespace iconvwrapper
 {
diff -urNad vdr-plugin-burn-0.1.0~pre21~/menuburn.h vdr-plugin-burn-0.1.0~pre21/menuburn.h
--- vdr-plugin-burn-0.1.0~pre21~/menuburn.h	2006-09-16 20:33:36.000000000 +0200
+++ vdr-plugin-burn-0.1.0~pre21/menuburn.h	2008-06-10 20:08:12.000000000 +0200
@@ -14,6 +14,7 @@
 #include <utility>
 #include <string>
 #include <vdr/menuitems.h>
+#include <limits>
 
 class cRecording;
 
diff -urNad vdr-plugin-burn-0.1.0~pre21~/menuitems.h vdr-plugin-burn-0.1.0~pre21/menuitems.h
--- vdr-plugin-burn-0.1.0~pre21~/menuitems.h	2006-09-16 20:33:36.000000000 +0200
+++ vdr-plugin-burn-0.1.0~pre21/menuitems.h	2008-06-10 20:08:12.000000000 +0200
@@ -20,6 +20,7 @@
 #include <vector>
 #include "boost/bind.hpp"
 #include <vdr/menuitems.h>
+#include <limits>
 
 namespace vdr_burn {
 namespace menu {
diff -urNad vdr-plugin-burn-0.1.0~pre21~/proctools/logger.cc vdr-plugin-burn-0.1.0~pre21/proctools/logger.cc
--- vdr-plugin-burn-0.1.0~pre21~/proctools/logger.cc	2006-09-16 17:22:02.000000000 +0200
+++ vdr-plugin-burn-0.1.0~pre21/proctools/logger.cc	2008-06-10 20:08:12.000000000 +0200
@@ -6,6 +6,7 @@
 #include <sstream>
 #include <algorithm>
 #include <cerrno>
+#include <string.h>
 
 namespace proctools
 {
